server/odc-core/src/main/resources/i18n/ErrorMessages.properties

```diff
@@ -126,6 +126,8 @@ com.oceanbase.odc.ErrorCodes.ObCreatePlRunPackageFailed=Create PL run package fa
 com.oceanbase.odc.ErrorCodes.ObGetExecuteDetailFailed=Get execute info failed, details: {0}
 com.oceanbase.odc.ErrorCodes.ObGetPlanExplainFailed=Get plan explain failed, may invalid sql or DDL, details: {0}
 com.oceanbase.odc.ErrorCodes.ObGetPlanExplainEmpty=Cannot get plain explain, may invalid sql or DDL, or some unsupported command, details: {0}
+com.oceanbase.odc.ErrorCodes.ObGetFullLinkTraceFailed=Get full link trace failed, please check if ob_enable_show_trace is enabled
```

Comment:
please check if ob_enable_show_trace is enabled

-->

please check if session variable 'ob_enable_show_trace' is 'ON'

Reply:
yes, updated this